openapi-spec/swagger.yaml

```diff
@@ -839,7 +945,7 @@ paths:
               schema:
                 type: array
                 items:
-                  $ref: '#/components/schemas/VolumeSpec'
+                  $ref: '#/components/schemas/VolumeRespSpec'
         401:
           description: NotAuthorized
```

Comment:
where is the authorization in the request ?

Reply:
Good point. Currently, we don't have header in our swagger file. Header may contain x-auth-token for authorization. Which we need to add in all API SPEC. We will add it.